fastapi/dependencies/models.py

```diff
@@ -33,6 +33,7 @@ def __init__(
         security_scopes_param_name: Optional[str] = None,
         security_scopes: Optional[List[str]] = None,
         use_cache: bool = True,
+        lifetime: str = "request",
```

Comment:
Why not add a `Literal["request", "app"]` here?

Reply:
I made an alias `DependencyLifetime = Literal["request", "app"]`